defi/src/protocols/data3.ts

```diff
@@ -39233,7 +39233,8 @@ const data3: Protocol[] = [
       "Leverage farming protocol on base",
     chain: "Base",
     logo: `${baseIconsUrl}/arcadia-v2.jpg`,
-    audits: "2",
+    audits: "5",
```

Comment:
keep 2 here please
0 -> No audits
1 -> Part of this protocol may be unaudited
2 -> Yes
3 -> This protocol is a fork of an existing audited protocol

Reply:
Thanks for the info, adjusted in commit